oauth2_http/javatests/com/google/auth/oauth2/JwtClaimsTest.java

```diff
@@ -31,19 +31,19 @@
 
 package com.google.auth.oauth2;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
nit: we should avoid star imports and keep individual imports

Reply:
Ack!